src/components/atoms/UiCheckbox/UiCheckbox.vue

```diff
@@ -70,8 +70,8 @@ import UiText from '../UiText/UiText.vue';
 import useInput from '../../../composable/useInput';
 import { keyboardFocus as vKeyboardFocus } from '../../../utilities/directives';
 
-export type CheckboxValue = string | object;
-export type CheckboxModelValue = boolean | object;
+export type CheckboxValue = string | Record<string, unknown>[];
```

Comment:
Q: Why is there an `unknown`? ;) Maybe it'd be nice to make usage of generic types here? 🤔 :) 
Something like this:
`type CheckboxValue<T> = string | Record<string, T>[];` 
WDYT? 🙂